L1Trigger/Phase2L1ParticleFlow/interface/PatternFile.h

```diff
@@ -0,0 +1,123 @@
+#ifndef L1Trigger_Phase2L1ParticleFlow_PatternFile_h
+#define L1Trigger_Phase2L1ParticleFlow_PatternFile_h
```

Comment:
Since this is only a pattern file for tracks and not for any other input or output formats, renaming it would be better to avoid confusion (plus in principle it could also be put in some track trigger package instead of here).

Reply:
Eventually the code will be generalized more. This is just a first step.